practitioner-service/src/test/java/uk/ac/ox/ndph/mts/practitioner_service/repository/HapiFhirRepositoryTests.java

```diff
@@ -57,53 +60,47 @@ void TestHapiRepository_WhenSavePractitioner_SendsBundleWithTransactionType() th
     }
 
     @Test
-    void TestHapiRepository_WhenSavePractitioner_ReturnsCorrectId() throws FhirServerResponseException {
+    void createPractitioner_WhenSavePractitioner_ReturnsCorrectId() throws FhirServerResponseException {
         // Arrange
         var responseBundle = new Bundle();
         when(fhirContextWrapper.executeTransaction(any(Bundle.class))).thenReturn(responseBundle);
-        when(fhirContextWrapper.toListOfResources(any(Bundle.class))).thenReturn(List.of(new Practitioner()));
-        var practitioner = new Practitioner();
-        practitioner.setId("123");
+        Practitioner RESPONSE_PRACTITIONER = new Practitioner();
```

Comment:
We currently don't trigger checkstyle on test classes but this naming convention for local vars will not pass. Why do you use it?